app/src/main/java/com/arygm/quickfix/ui/elements/QuickFixDetailsdescription.kt

```diff
@@ -99,58 +100,116 @@ fun QuickFixDetailsScreen(
                 }
           }
 
-          // Description Section with "Show More"
-          Spacer(modifier = Modifier.height(screenHeight * 0.02f))
+          // Description Section
+          Spacer(modifier = Modifier.height(8.dp))
           Text(
               text = "Description",
               style = poppinsTypography.headlineMedium,
               color = MaterialTheme.colorScheme.onBackground,
-              modifier = Modifier.padding(bottom = screenHeight * 0.01f))
-          Column {
-            Text(
-                text = if (isExpanded) quickFix.title else quickFix.title.take(250) + "...",
-                style = poppinsTypography.bodyMedium,
-                color = MaterialTheme.colorScheme.onSurface,
-            )
-            TextButton(
-                onClick = { onShowMoreToggle(!isExpanded) },
-                modifier = Modifier.align(Alignment.CenterHorizontally)) {
-                  Text(
-                      text = if (isExpanded) "Show less" else "Show more",
-                      style = poppinsTypography.labelMedium,
-                      color = MaterialTheme.colorScheme.primary)
-                }
-          }
+              modifier = Modifier.padding(bottom = 8.dp))
+          // Show the full description or truncate it based on isExpanded
+          Text(
+              text =
+                  if (isExpanded) quickFix.title
+                  else quickFix.title.take(250) + if (quickFix.title.length > 250) "..." else "",
```

Comment:
One possible suggestion here would be to specify the choice of 250, maybe by creating a specific constant.

Reply:
It was already decided with team to take the value 250